.npmignore

```diff
@@ -1,8 +0,0 @@
-src
```

Comment:
Why is this file deleted? we can ignore these files when we publish it to npm to reduce the bundle size.

Reply:
I've whitelisted the `dist/**/*` directory using the `files` key on `package.json`, that way it's easier to manage the package rather than blacklisting any new files we add. I did a dry run of `npm publish` and it seems to be packaging everything we require as expected.

https://blog.npmjs.org/post/165769683050/publishing-what-you-mean-to-publish